motor/providers/awsec2ebs/provider.go

```diff
@@ -199,6 +199,8 @@ func (p *Provider) Close() {
 				log.Error().Err(err).Msg("unable to delete volume")
 			}
 		}
+	} else {
```

Comment:
Since the default behaviour is now that we do not delete the volume, I would add an info log, once the volume was deleted.

Reply:
Done.